DeviceAPICharter.html

```diff
@@ -470,6 +470,7 @@ <h3 id="history">Charter history</h3>
                   31 December 2017
                 </td>
                 <td>
+                  Rechartered under the name "Device and Sensors Working Group". Added Fuqiao Xue as team contact in September 2017. Chair changed from Anssi Kostiainen to Frederick Hirsch in September 2017.

```

Comment:
s/from Anssi Kostiainen to Frederick Hirsch/from Frederick Hirsch to Anssi Kostiainen/

Reply:
Oops. Will push a fix for that.